src/extension.ts

```diff
@@ -338,6 +344,18 @@ export async function activate(context: vscode.ExtensionContext) {
             );
         }
     );
+    const clueOpenSimulator: vscode.Disposable = vscode.commands.registerCommand(
+        "deviceSimulatorExpress.clue.openSimulator",
+        () => {
+            telemetryAI.trackFeatureUsage(
+                TelemetryEventName.MICROBIT_COMMAND_OPEN_SIMULATOR
+            );
+            telemetryAI.runWithLatencyMeasure(
+                openClueWebview,
+                TelemetryEventName.MICROBIT_PERFORMANCE_OPEN_SIMULATOR
+            );
```

Comment:
I think we should just make it run openClueWebview() for now and have a task to add telemetry later instead of using microbit telemetry.